package.json

```diff
@@ -32,7 +32,7 @@
     "lint:scss:fix": "prettier --write --ignore-path ./.gitignore **/*.scss",
     "lint:scss:prod": "prettier -c --end-of-line lf --ignore-path ./.gitignore **/*.scss",
     "start": "npm run watch",
-    "test": "npm run test:spec && npm run test:e2e",
+    "test": "cross-env-shell SASS_PATH=node_modules \"stencil test --spec --e2e\"",
```

Comment:
`npm run test:spec && npm run test:e2e` is the same as running `stencil test --spec && cross-env-shell SASS_PATH=node_modules \"stencil test --e2e\"`, which is calling `stencil test` twice, once for `spec`-files and once for `e2e`-files. That's unnecessary, since we can just call `stencil test` once, for both `spec`- and `e2e`-files, which shaves off a handful of seconds because Stencil doesn't have to start things up twice 😊